HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectType.cs

```diff
@@ -9,32 +9,68 @@ namespace GraphicalEditor.Models.MapObjectRelated
     public class MapObjectType
     {
         public TypeOfMapObject TypeOfMapObject { get; set; }
+
+        public MapObjectType()
+        { 
+        }
+   
```

Comment:
Why empty method?

Reply:
this is constructor

Reply:
Okay, but why empty?

Reply:
it is default constructor

Reply:
l need default so i can create object

Reply:
If your class doesn't need to do anything in the default constructor - don't define it. The compiler will generate one for you.

Reply:
but if it has constructor with parametars we need explicit to write and the default constructor

Reply:
You are right if you keep constructors with parameters. My suggestion is to move all constructors if you do not use them. But like you said, you will use them, so if it becomes true, then it is okay for me.